setup.py

```diff
@@ -55,6 +55,9 @@ def build(self):
         # remove java source as it doesn't need to be packaged
         shutil.rmtree(os.path.join(self.build_dir, "planner"))
 
+        # copy jar to source directory so dask-sql can be imported from there
+        self.copy_tree(os.path.join(self.build_dir, "dask_sql/jar"), "dask_sql/jar")
```

Comment:
Maybe we can check the [`READTHEDOCS` environment variable]( https://docs.readthedocs.io/en/stable/builds.html#build-environment ) before doing this?

Reply:
Great idea! Was considering setting some sort of RTD flag but didn't realize there was one in place 🙂

Reply:
I'm pretty sure this should be okay as an RTD-only fix - the niche case it would cover is allowing users to import from within the source repo if they use `python setup.py install` (this wouldn't work for pip installs since those copy the source directory to a temp folder), and I would imagine the majority of users / developers working directly within the source repo would probably want an editable install where this isn't an issue.

If anyone opens an issue expressing difficulty with that specific scenario, it shouldn't be an issue to make it a general fix